### ISSUE TITLE
[Movecraft] Task #97 generated an exception?

### ISSUE THREAD

**Issue by HightechCaveman** [eirikh1996/Movecraft#58]
**Describe the bug**
Movecraft broke, probably by me. Players were able to sail perfectly fine, but it suddenly breaks after a while.

**Proof**
03.07 11:45:43 [Server] INFO java.lang.ClassCastException: null
03.07 11:45:43 [Server] WARN [Movecraft] Task #97 for Movecraft v7.0.0_for_1.8.8-1.16.1_beta_26 generated an exception
03.07 11:45:43 [Server] INFO java.lang.ClassCastException: null
03.07 11:45:43 [Server] WARN [Movecraft] Task #97 for Movecraft v7.0.0_for_1.8.8-1.16.1_beta_26 generated an exception
03.07 11:45:43 [Server] INFO java.lang.ClassCastException: null
https://pastebin.com/W7X1tMCX
It just repeats that over and over, and I think it has been for the last 8 hours I was sleeping.

**Versions (please complete the following information):**
 - Movecraft: Latest 
 - Java:
 - Minecraft: 1.14.4

**Additional context**
I have the Cannons plugin, and the most recent update for movecraft if any of them cause problems. I can always downgrade back to the last version.

**Comment by SydMontague** [comment]
Not sure if it's the same issue but we got these exception spammed:
```[22:49:57] [Server thread/WARN]: [Movecraft] Task #53 for Movecraft v7.0.0_for_1.8.8-1.16.1_beta_26 generated an exception
java.lang.ClassCastException: class java.lang.Byte cannot be cast to class org.bukkit.block.data.BlockData (java.lang.Byte is in module java.base of loader 'bootstrap'; org.bukkit.block.data.BlockData is in unnamed module of loader 'app')
	at net.countercraft.movecraft.async.AsyncManager.processFadingBlocks(AsyncManager.java:950) ~[Movecraft.jar:?]
	at net.countercraft.movecraft.async.AsyncManager.run(AsyncManager.java:1089) ~[Movecraft.jar:?]
	at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:81) ~[Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:394) [Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at net.minecraft.server.v1_15_R1.MinecraftServer.b(MinecraftServer.java:1036) [Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:406) [Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:984) [Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:824) [Spigot.jar:git-Spigot-ae72bf4-55b6be3]
	at java.lang.Thread.run(Thread.java:834) [?:?]```

**Comment by UAVwaffle** [comment]
try removing all redstone components on a ship and see if that works,  I was getting a similar bug whenever I had redstone on my crafts.   Here is my bug report probs have a look at it to see if it's the same issue: https://github.com/eirikh1996/Movecraft/issues/60

**Comment by eirikh1996** [comment]
Fixed in 04276145bff4d2d8c24902dde3feeab3207bea96 and ea91b716681f3e442eff012f0c1d2568520cce3a
